pom.xml

```diff
@@ -179,6 +179,11 @@
         <artifactId>mockito-core</artifactId>
         <version>${mockito.version}</version>
       </dependency>
+      <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-auth</artifactId>
+        <version>0.8.4</version>
```

Comment:
This should still be `${dropwizard.version}`. `dropwizard.version` is equal to 0.8.4, we don't want to have to change it in two places if we want to upgrade versions in the future.